ampersand-model.js

```diff
@@ -20,6 +24,30 @@ var wrapError = function (model, options) {
     };
 };
 
+// recursive function to find the attrs in the serialized object
+function transformForPatch(obj, attrs) {
```

Comment:
Yes this is what I meant, thanks.  I should have been more explicit, sorry for the confusion.